examples/assign-or-extend-a-task/Ballerina.toml

```
@@ -0,0 +1,10 @@
+[package]
+org = "chathura"
```

Change the org to WSO2
```suggestion
org = "wso2"
```

---

done